### ISSUE TITLE
Update to new Facepunch.Steamworks version?

### ISSUE THREAD

**Issue by LoneSurvivor82** [momower1/SteamNetworking#1]
Hi,

your code seems to use an outdated version of Facepunch.Steamworks. Do you plan to upgrade your code to use the current version?

**Comment by miles5600** [comment]
yeah I have that issue too maybe ask him in the Facepunch discord he is in that discord every day so you can ask him.

**Comment by miles5600** [comment]
BTW he updated the api so is that your issue?
what issue are you having

**Comment by momower1** [comment]
Sorry, I do not plan to update this anytime soon.
But if someone wants to make a pull request with the newer version, I'm willing to merge it.
